src/screens/Settings/components/OtherSettings/index.tsx

```diff
@@ -211,14 +211,14 @@ export default function OtherSettings({
           </span>
           {isLinuxNative && (
             <span className="setting">
-              <span className={classNames('toggleWrapper', { isRTL: isRTL })}>
+              <label className={classNames('toggleWrapper', { isRTL: isRTL })}>
                 <ToggleSwitch
```

Comment:
Why changing from a span to a label fixes it?
Also, would be good to check the logic on the backend as well. To be sure it's using the right path.

Reply:
I see, ok then. 👍🏽